scripts/update-unicode.py

```diff
@@ -1228,7 +1228,7 @@ def stats(self, int_size) -> Stats:
             offsets2_int_size=0,
         )
 
-    @classmethod
+    @staticmethod
     def test(cls):
```

Comment:
huh, I thought `staticmethod` doesn't take a `cls` argument, does it? If there's any callers they should now fail with "too few arguments"?

Reply:
You are right. Fixed in #534